src/main/webapp/editor/blocklyc.jsp

```diff
@@ -32,6 +34,9 @@
         <script type="text/javascript" src="<url:getCdnUrl url="/blocklyc.js"/>"></script>
         <script type="text/javascript" src="<url:getCdnUrl url="/utils.js"/>"></script>
         <script type="text/javascript" src="<url:getCdnUrl url="/editor.js"/>"></script>
+
+        <link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css" />
```

Comment:
Same thing with the CSS file. Please load up the desired version into the webapp/cdn/lib/ folder.